java/ql/src/Security/CWE/CWE-200/AndroidWebViewSettingsContentAccess.ql

```diff
@@ -0,0 +1,21 @@
+/**
+ * @name Android WebSettings content access
+ * @description Access to content providers in a WebView can enable JavaScript to access protected information.
+ * @kind problem
+ * @id java/android-websettings-content-access
+ * @problem.severity warning
+ * @security-severity 6.5
+ * @precision high
+ * @tags security
+ *       external/cwe/cwe-200
+ */
+
+import java
+import semmle.code.java.frameworks.android.WebView
+
+from MethodAccess ma
+where
+  ma.getMethod() instanceof AllowContentAccessMethod and
```

Comment:
This is enabled by default, right? I think the query should take that into consideration and use dataflow to identify `WebSettings` instances that _don't_ set this to `false` explicitly.